.github/workflows/extra_tests.yml

```diff
@@ -63,10 +70,36 @@ jobs:
           key: cargo-registry-${{ hashFiles('shadow/src/Cargo.lock') }}
           restore-keys: |
             cargo-registry-
+      - name: Restore sccache cache
+        uses: actions/cache@v3
+        continue-on-error: false
+        with:
+          path: ${{ env.SCCACHE_DIR }}
+          key: sccache-${{ env.CONTAINER }}-${{ env.CC }}-${{ env.BUILDTYPE }}-${{ hashFiles('shadow/src/Cargo.lock') }}
+          restore-keys: |
+                        sccache-${{ env.CONTAINER }}-${{ env.CC }}-${{ env.BUILDTYPE }}-
```

Comment:
Will this cause the cache size to always increase? As we update dependency versions, will it continue to keep build artifacts from old dependencies since we never "reset" the cache?

Reply:
Dug into this a bit; I think you're right. I'll add the month back into the cache key so it gets reset monthly.